### ISSUE TITLE
Feature request: Implement confidence score visualisation

### ISSUE THREAD

**Issue by menelic** [hayabhay/whisper-ui#51]
This whisper gui implementation visualizes the confidence score by coloring words: https://github.com/jojojaeger/whisper-streamlit This would be helpful for quicker review or for model accuracy comparison

**Comment by bi1101** [comment]
yes, that would be very nice. I'm a teacher and using this to assist scoring students recording would be fantastic

**Comment by hayabhay** [comment]
Bumping this up as a feature for the upcoming release with the [new version](https://github.com/hayabhay/frogbase/discussions/56).
